src/common/components/ConditionalLink.js

```diff
@@ -0,0 +1,11 @@
+import { Link } from 'react-router-dom';
+
+const ConditionalLink = ({ children, to, condition }) => {
+  if (!!condition && to) {
+    return <Link to={to}>{children}</Link>;
+  } else {
+    return <>{children}</>;
+  }
```

Comment:
Suggestion: This feels easier to read for me
```suggestion
  return (!!condition && to)
    ? <Link to={to}>{children}</Link>
    : <>{children}</>;
```

Reply:
The [Stack Overflow answer](https://stackoverflow.com/a/56278522) had a ternary, but I found it to be overly dense/harder to follow. Spreading it out is more readable and will make debugging easier.